### DISCUSSION TITLE
似乎和hexo-log@4.0.1版本有冲突

### DISCUSSION THREAD

**Discussion by zhutmost** [ppoffice/hexo-theme-icarus#1146] [General]
更新了hexo-log这个包之后，hexo g会失败，报错如下：

<img width="974" alt="image" src="https://user-images.githubusercontent.com/20497382/201337203-b116eeeb-1585-4b6b-ba16-4240a082896a.png">

**Comment by ppoffice** [comment]
@zhutmost 除了升级hexo-log以外你还做了其他改动么？

**Comment by zhutmost** [comment]
没有。我反复测试了，降级这个包到3.2.0就没问题，升级到4.0.1后hexo g就报错

**Reply by ppoffice** [reply]
我这边无法复现此问题。请检查icarus的layout文件是否被修改过。另外其他的依赖有改动过么？最好能分享你博客的源代码方便debug。最后，升级hexo-log的理由是什么？

**Reply by zhutmost** [reply]
没啥理由，只是我看了下它的release notes，没提到啥特别的，就顺手更新了下。

**Comment by zhutmost** [comment]
<img width="826" alt="image" src="https://user-images.githubusercontent.com/20497382/201462020-6ecb9508-17a9-437d-8a85-9751cd37d5ca.png">
这是我的所有安装的包，我确认我只变化了我的hexo-log这一个依赖的版本。如果您无法复现的话，就先把这个discussion搁着吧，看看过两天会不会有新的hexo-log更新，或者观察下其他人有没有类似的问题
